server/main.lua

```diff
@@ -131,23 +131,19 @@ RegisterNetEvent('hospital:server:TreatWounds', function(playerId)
 	local src = source
 	local player = exports.qbx_core:GetPlayer(src)
 	local patient = exports.qbx_core:GetPlayer(playerId)
-	if player.PlayerData.job.name ~= "ambulance" or not patient then return end
+	if player.PlayerData.job.name ~= "ambulance" or not patient or not exports.ox_inventory:RemoveItem(src, 'bandage', 1) then return end
 
-	player.Functions.RemoveItem('bandage', 1)
-	TriggerClientEvent('inventory:client:ItemBox', src, exports.ox_inventory:Items()['bandage'], "remove")
-	TriggerClientEvent("hospital:client:HealInjuries", patient.PlayerData.source, "full")
+	lib.callback("qbx_medical:client:heal", patient.PlayerData.source, false, "full")
 end)
 
 ---@param playerId number
 RegisterNetEvent('hospital:server:RevivePlayer', function(playerId)
 	if GetInvokingResource() then return end
-	local player = exports.qbx_core:GetPlayer(source)
-	local patient = exports.qbx_core:GetPlayer(playerId)
+	local src = source
+	local player = exports.qbx_core:GetPlayer(playerId)
```

Comment:
player seems to only be used to get playerId anyway, so this could be simplified further not to convert to player.